pyproject.toml

```diff
@@ -40,7 +39,7 @@ source = [
 ]
 
 [tool.coverage.report]
-fail_under = 69
+fail_under = 67
```

Comment:
Not sure what the point is of asserting on coverage if we're just going to keep bumping the threshold down. It may not be worth writing tests for full coverage, but in that case we don't get any value from measuring it.

Reply:
Yeah, this is the worst part of <100% coverage.  My next PR bumps it up to 73 if that helps…?